build-utils/webpack.prod.js

```diff
@@ -0,0 +1,38 @@
+const commonPaths = require('./common-paths')
+const webpack = require('webpack')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+
+const config = {
+  mode: 'production',
+  output: {
+    filename: 'static/[name].[hash].js'
+  },
+  devtool: 'source-map',
+  module: {
+    rules: [
+      {
+        test: /\.less$/,
+          use: [
+            {
+              // We configure 'MiniCssExtractPlugin'              
+              loader: MiniCssExtractPlugin.loader,
```

Comment:
if we add this a seperate one here would it work (and move the other style loaders to common:
i.e.

does this work ?

```JSON
 "rules": [
  ...
    {
    "test": /\.css$/,
    "use": [MiniCssExtractPlugin.loader, 'css-loader']
    }
   {
    "test": /\.less$/,
    "use": [
     {
      "options": {
       "importLoaders": 2
      }
     },
     "postcss-loader",
     "less-loader"
    ]
   }

```

Also was wondering why we need postcss ?

Reply:
I was able to build like that:
```
rules: [
      {
        test: /\.css$/,
        use: [MiniCssExtractPlugin.loader, 'css-loader']
      },
      {
        test: /\.less$/,
        use: [
          {
            loader: 'css-loader',
            options: {
              importLoaders: 2,
            }
          },
          'postcss-loader',
          'less-loader'
        ]
      }
```
But now the bundled `app.js` is huge and no `.css` was created:
<img width="769" alt="Screen Shot 2020-09-17 at 10 38 48 AM" src="https://user-images.githubusercontent.com/16461988/93399674-28d3a100-f8d2-11ea-9ed9-ebd5f3363097.png">

I'll make a quick research to answer this question:
> Also was wondering why we need postcss ?

Reply:
Ok. I have removed the `postcss` from the Webpack. It seems useless for now. We can change this if we need later on. This is the bundle result that I'm getting after changing the code to this:
```
{
        test: /\.less$/,
        use: [
          {
            loader: MiniCssExtractPlugin.loader
          },
          'css-loader',
          'less-loader'
        ]
      }
```
<img width="733" alt="Screen Shot 2020-09-17 at 11 20 11 AM" src="https://user-images.githubusercontent.com/16461988/93402040-c7163580-f8d7-11ea-9efd-370ddc404106.png">